kusama-guide/static/css/custom.css

```diff
@@ -117,119 +117,29 @@ body {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #ddd;
+  font-weight: 700;
 }
-.github-fork-ribbon:active,
+
 .github-fork-ribbon:hover {
-  background-color: rgba(0, 0, 0, 0);
-}
-.github-fork-ribbon:after,
-.github-fork-ribbon:before {
-  position: absolute;
-  display: block;
-  width: 15.38em;
-  height: 1.54em;
-  top: 3.23em;
-  right: -3.23em;
-  -webkit-box-sizing: content-box;
-  -moz-box-sizing: content-box;
-  box-sizing: content-box;
-  -webkit-transform: rotate(45deg);
-  -moz-transform: rotate(45deg);
-  -ms-transform: rotate(45deg);
-  -o-transform: rotate(45deg);
-  transform: rotate(45deg);
-}
-.github-fork-ribbon:before {
-  content: "";
-  padding: 0.38em 0;
-  background-color: #e6007a;
-  background-image: -webkit-gradient(
-    linear,
-    left top,
-    left bottom,
-    from(rgba(0, 0, 0, 0)),
-    to(rgba(0, 0, 0, 0.15))
-  );
-  background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.15));
-  background-image: -moz-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.15));
-  background-image: -ms-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.15));
-  background-image: -o-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.15));
-  background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.15));
-  -webkit-box-shadow: 0 0.15em 0.23em 0 rgba(0, 0, 0, 0.5);
-  -moz-box-shadow: 0 0.15em 0.23em 0 rgba(0, 0, 0, 0.5);
-  box-shadow: 0 0.15em 0.23em 0 rgba(0, 0, 0, 0.5);
-  pointer-events: auto;
-}
-.github-fork-ribbon:after {
-  content: attr(data-ribbon);
-  color: #fff;
-  font: 700 1em "Helvetica Neue", Helvetica, Arial, sans-serif;
-  line-height: 1.54em;
-  text-decoration: none;
-  text-shadow: 0 -0.08em rgba(0, 0, 0, 0.5);
-  text-align: center;
-  text-indent: 0;
-  padding: 0.15em 0;
-  margin: 0.15em 0;
-  border-width: 0.08em 0;
-  border-style: dotted;
-  border-color: #fff;
-  border-color: rgba(255, 255, 255, 0.7);
-}
-.github-fork-ribbon.left-bottom,
-.github-fork-ribbon.left-top {
-  right: auto;
-  left: 0;
-}
-.github-fork-ribbon.left-bottom,
-.github-fork-ribbon.right-bottom {
-  top: auto;
-  bottom: 0;
-}
-.github-fork-ribbon.left-bottom:after,
-.github-fork-ribbon.left-bottom:before,
-.github-fork-ribbon.left-top:after,
-.github-fork-ribbon.left-top:before {
-  right: auto;
-  left: -3.23em;
-}
-.github-fork-ribbon.left-bottom:after,
-.github-fork-ribbon.left-bottom:before,
-.github-fork-ribbon.right-bottom:after,
-.github-fork-ribbon.right-bottom:before {
-  top: auto;
-  bottom: 3.23em;
-}
-.github-fork-ribbon.left-top:after,
-.github-fork-ribbon.left-top:before,
-.github-fork-ribbon.right-bottom:after,
-.github-fork-ribbon.right-bottom:before {
-  -webkit-transform: rotate(-45deg);
-  -moz-transform: rotate(-45deg);
-  -ms-transform: rotate(-45deg);
-  -o-transform: rotate(-45deg);
-  transform: rotate(-45deg);
-}
-/*# sourceMappingURL=gh-fork-ribbon.min.css.map */
+  color: white !important;
```

Comment:
Should we also update the Kusama guide's Contributing ribbon too?

Reply:
It is updated

Reply:
Ah I see it now - i forgot to run `yarn kusama:start`. Looks all good!